packages/smoldot-provider/src/FsDatabase.ts

```diff
@@ -25,9 +25,9 @@ export class FsDatabase implements Database {
   load(): string {
     try {
       statSync(this.#path);
-    } catch (error: any) { 
+    } catch (error: any) { // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
You should not disable linter. There are error types that can be used. Any should not be used

Reply:
Yeah missed this one I can use `NodeJS.ErrnoException`

Reply:
oh no it cant .. you aren't allowed to specify error types in catch blocks.  Typescript won't allow it.  I've changed it to unkown instead, that's better. 

see https://stackoverflow.com/questions/42618089/how-do-you-use-typed-errors-in-async-catch#42618403